.orchestra/config/components/toolchain/lib/gdb.lib.yml

```diff
@@ -42,22 +43,47 @@ configure: |
     --disable-gold \
     --disable-gprof \
     --disable-ld \
-    --disable-gdbserver \
+    --enable-gdbserver \
     --enable-64-bit-bfd \
     --disable-install-libbfd \
     --disable-install-libiberty \
     --without-guile \
     --with-system-readline \
     --enable-nls \
-    --enable-tui=no \
+    --enable-tui \
     --with-mpfr="$ORCHESTRA_ROOT" \
     --without-included-gettext \
     --disable-shared \
     --enable-obsolete \
     --without-stage1-ldflags \
+    --with-python="$ORCHESTRA_ROOT/bin/python" \
+    --with-system-gdbinit="$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit" \
     CFLAGS="-w -ggdb3 -O3" \
     CXXFLAGS="-w -ggdb3 -O3"
-build_system: make
+install: |
+  cd "$BUILD_DIR"
+  (@= make @)
+  (@= make @) install DESTDIR="$DESTDIR"
+
+  GDBINIT="$DESTDIR$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit"
+  mkdir -p "$(dirname "$GDBINIT")"
+
+  cat >"$GDBINIT" <<'EOF'
+  set pagination off
+  EOF
+
+  (@- if triple.startswith("x86_64"): @)
+  echo "set disassembly-flavor intel" >> "$GDBINIT"
+  (@ end @)
+
+dependencies:
+  - expat
+  - gmp
+  - ncurses
+  - python
+  - readline
+  - source-highlight
+  - toolchain/host/gcc
```

Comment:
Please use `host-c-toolchain`. One day we might drop `gcc` as an host toolchain.

Reply:
It doesn't need the compiler (not as a non-build dependency at least), it needs `gcc-libs` and this component provides them.

Reply:
Uhm, I wonder why. Can you link the archlinux package you got insipration from?

Reply:
It needs `libstdc++` which is provided by that binary archive.

https://github.com/archlinux/svntogit-packages/blob/packages/gdb/trunk/PKGBUILD#L49

```
$ pacman -Qo /usr/lib/libstdc++.so.6
/usr/lib/libstdc++.so.6 is owned by gcc-libs 12.2.0-1
```

I guess we can statically link to it though.

Reply:
Ah no it also uses `libgcc_s.so.1`